module/x/gravity/types/msgs.go

```diff
@@ -549,8 +549,9 @@ func (b *MsgLogicCallExecutedClaim) ClaimHash() []byte {
 }
 
 // NewMsgSetOrchestratorAddress returns a new msgSetOrchestratorAddress
-func NewMsgCancelSendToEth(val sdk.ValAddress, id uint64) *MsgCancelSendToEth {
+func NewMsgCancelSendToEth(user sdk.AccAddress, id uint64) *MsgCancelSendToEth {
```

Comment:
Slightly confused, this message isnt registered on the handler, how is this suppose to be executed?

Reply:
lol it won't. I noticed it and fixed on my PR

Reply:
corrected this here.